packages/dashboard/src/components/templates/platforms/platform-template.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import Frame from 'Components/elements/frame';
+import { useStores } from 'Stores';
+
+const AboutUs: React.FC = () => {
```

Comment:
Is this a temporary file?

Reply:
yes, its a temporary file, will delete it 🙏